src/datasource-zabbix/zabbix/connectors/zabbix_api/zabbixAPIConnector.ts

```diff
@@ -132,18 +132,6 @@ export class ZabbixAPIConnector {
     return this.request('event.acknowledge', params);
   }
 
-  getGroupsWithHosts() {
-    // Adding random limit to get past Zabbix server cache, this query is cached in datasource instead
-    const randomLimit = Math.floor(Math.random() * 1000) + 1000;
-    const params = {
-      selectHosts: ['hostid', 'name', 'maintenance_status'],
-      output: ['groupid', 'name'],
-      limit: randomLimit
-    };
-
-    return this.request('hostgroup.get', params);
-  }
```

Comment:
Code search seems to show same function name is used elsewhere in grafana-zabbix: https://github.com/search?q=repo%3Adigiaiiris%2Fgrafana-zabbix%20getGroupsWithHosts&type=code

Are you sure this can be deleted?

Reply:
Or should those places be changed or cleaned up as well?

Reply:
Sorry, made the PRs in wrong order. Meant to do them simultaneously but had some hazzle.
The usage has been removed here: https://github.com/digiaiiris/iiris-grafana-app/pull/219/

Reply:
What about references in *this* project? I was referring to those.

Reply:
Ouch, I saw those references earlier but forgot them totally. Good point, thanks. Fixed in 7ff36aab36b8eda22badeae18894b7f4ce2032b0